src/ui/public/styles/bootstrap/variables.less

```diff
@@ -13,6 +13,7 @@
 @gray:                   lighten(@gray-base, 33.5%); // #555
 @gray-light:             lighten(@gray-base, 46.7%); // #777
 @gray-lighter:           lighten(@gray-base, 93.5%); // #eee
+@gray-lightest:          lighten(@gray-base, 98%);
```

Comment:
Basically this stuff.

Reply:
So there's actually a [_custom_variables.less](https://github.com/elastic/kibana/pull/27510/files/a1ebee690a20d4feb49b8b52f3ee4296c62ff5f3#diff-0736fbf0569a87ab26afa9a4841aa1b0) file that does override the base bootstrap vars.

Reply:
Got it. OK. Thanks, that make total sense now.